### ISSUE TITLE
Add an mjml root tag to separate the content and the meta definition

### ISSUE THREAD

**Issue by arnaudbreton** [mjmlio/mjml#78]
Based on coming new features we want to add in MJML (#79, #80, #81), it would make things clearer to define an `mjml` root tag, along with an `mj-head` tag in order to separate the content, currently defined in `mj-body` and meta definition.  This issue represents an implementation we think will cover this case nicely. The idea here is to discuss openly here and to settle on a final implementation by the end of the week.

The new syntax would be the following:
```html
<mjml>
  <mj-head>
    <mj-attributes>
      ...
    </mj-attributes>
    <mj-style>
      ...
    </mj-style>
    <mj-raw>
      ...
    </mj-raw>
  </mj-head>
   <mj-body>
      ...
   </mj-body>
</mjml>
```

To remain backward compatible, we would make this tag optional, so a currently valid MJML document would remain valid and keep the markup light for users who don't need such meta definition:
```html
   <mj-body>
      ...
   </mj-body>
```

**Comment by arnaudbreton** [comment]
`mjml` root tag is now available from 2.0.0. Closing.

**Comment by Marvin256** [comment]
I'm using MJML 3.0.1 and her's what I get when puting mj-raw in mj-head:

Warning: No handler found for: mj-raw, in mj-head, skipping it

**Comment by iRyusa** [comment]
@Marvin256 `mj-raw` isn't available yet in MJML cf #82 still open
